package.json

```diff
@@ -2,8 +2,9 @@
   "name": "tinlake",
   "version": "0.0.1",
   "description": "Centrifuge Tinlake Contracts Client",
-  "main": "src/index.js",
-  "module": "src/index.js",
```

Comment:
main and module should be setup to /build and we should something like rollup/webpack. We need to support both cjs and esm

Reply:
Fixed